cmd/run/run.go

```diff
@@ -185,6 +185,8 @@ func NewRunCommand() *cobra.Command {
 
 	flags.Uint32("max-concurrent-reads-for-check", defaultConfig.MaxConcurrentReadsForCheck, "the maximum allowed number of concurrent datastore reads in a single Check query. A high number will consume more connections from the datastore pool and will attempt to prioritize performance for the request at the expense of other queries performance.")
 
+	flags.Uint64("max-condition-evaluation-cost", defaultConfig.MaxConditionEvaluationCost, "defines the maximum cost for CEL condition evaluation before a request will return an error to the client")
```

Comment:
Please use the same description as the one provided in `.config-schema.json`. They need not be different.

side note - we've discussed constructing these flags off of the `.config-schema.json` in the future as well, so this is why we strive to keep them consistent relative to one another.